sonic_y_cable/y_cable_vendor_mapping.py

```diff
@@ -0,0 +1,13 @@
+mapping = {
+    "Credo": {
```

Comment:
Can we please make the vendor name and model names in lower cases, to comply with fetching vendor mapping attribute. 
example
mapping = {
    "credo": {
        "cacl2x321p2pa1ms": "credo.y_cable_credo"
    }
}

Reply:
Yes, we can convert the PN and vendor name in lower case.